slider/__main__.py

```diff
@@ -1,34 +1,34 @@
-import click
-
 from . import Library
 
+if __name__ == '__main__':
```

Comment:
Does this file get executed on a normal import, or only when used as an executable? I agree with the nicer import error, but I think we may be able to drop this branch.

Reply:
good catch, you're right.